Xamarin.Forms.Controls/GalleryPages/CarouselView/CarouselPageViewModel.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace Xamarin.Forms.Controls
+{
+    public class CarouselViewModel : INotifyPropertyChanged
+    {
+        public CarouselViewModel()
+        {
+			Source = Source ?? new ObservableCollection<int>() { 1 };
+        }
+
+        ObservableCollection<int> _Source;
+        /// <summary>
+        /// Images
+        /// </summary>
```

Comment:
@alexrainman Are these summaries needed? There are other examples in different files as well. Maybe we need to make them better or completely remove them?